hoodie-client/src/main/java/com/uber/hoodie/table/HoodieCopyOnWriteTable.java

```diff
@@ -313,6 +318,7 @@ private void assignInserts(WorkloadProfile profile) {
                 FSUtils.getFileId(filename));
             sf.sizeBytes = file.getFileSize();
             smallFileLocations.add(sf);
+            smallFileIds.add(FSUtils.getFileId(filename));
```

Comment:
may be some form of consolidation between the list returned by `getSmallFiles` and the `smallFileIds` member?

Reply:
Yeah, did that.